smart-contract-verifier/src/metrics.rs

```diff
@@ -23,4 +25,28 @@ lazy_static! {
         "contract compilation time in seconds",
     )
     .unwrap();
+    pub static ref COMPILE_THREADS_NUMBER: Gauge = register_gauge!(
```

Comment:
that's very misleading name
I firstly thought that this is just constant and is equal to the config value
let's call this `COMPILES_IN_FLIGHT` and add description of `number of compilations currently running`